include/swift-c/DependencyScan/DependencyScan.h

```diff
@@ -49,6 +49,8 @@ typedef struct {
 } swiftscan_string_set_t;
 
 typedef enum {
+  // This dependency info encodes two ModuleDependencyKind types:
```

Comment:
For now, preserve the format used by libSwiftScan. Once we teach the clients about the new dependency kind, we can then add it here.